robot/robot.go

```diff
@@ -158,6 +158,9 @@ type Robot interface {
 
 	// Version returns version information about the robot.
 	Version(ctx context.Context) (VersionResponse, error)
+
+	// ListTunnels returns information on available traffic tunnels.
+	ListTunnels() []config.TrafficTunnelEndpoint
```

Comment:
returning an object from the `config` package feels slightly weird, but don't have a better idea right now

Reply:
I agree... I think we need a config object, but I could certainly make a type that is identical to `TrafficTunnelEndpoint` to return to users. That has two downsides, though: we need (simple) conversion logic between the two and the fields will have to be kept in sync (there's also `trafficTunnelEndpointData` that has the real JSON struct tags and needs to be similary kept in sync.)